### ISSUE TITLE
What do you love about your community word clouds

### ISSUE THREAD

**Issue by jridener** [opensmc/opensmc.github.io#12]
OpenSMC has a Google Form asking County residents what they love about where they live and what could be improved about their communities: [(http://goo.gl/forms/mkXlEjxHhJ)]

The task is to take the responses: [(https://docs.google.com/spreadsheets/d/1XdUI2AyVjmd20q8AZlDdrct2d5TfyS3hYai0a6dHaZU/edit?usp=sharing)] and create one word cloud for the positive responses and one word cloud for the negative responses.

If you have problems accessing the form results from the link, please contact John at jridener@opensmc.org or @jridener on slack.

Next, create a page on the OpenSMC site where the two word clouds can be displayed. Include the form and a link to see the results.

**Comment by verythorough** [comment]
Word cloud webapp: https://www.jasondavies.com/wordcloud/

**Comment by nguyenvyc** [comment]
Here are the two word clouds, one for positive and the other for negative. Should I take out some words?
![wordcloud negative](https://cloud.githubusercontent.com/assets/16928487/14131161/87171342-f5ed-11e5-8d87-0f35af70e61d.png)
![wordcloud love](https://cloud.githubusercontent.com/assets/16928487/14131162/8726eea2-f5ed-11e5-977e-cb3116f86afb.png)

**Comment by verythorough** [comment]
@nguyenvyc, as I look closer, I see what you mean.  Yes, I agree that it makes sense to remove some words.  For example, city names (especially partial names) don't really make sense, and and functional words like still, near, since, and etc (literally!) don't really add any meaning.

I wonder if it makes sense to reduce each entry to specific key words, for example "People are willing to get involved" => "people" and "involved."  If there are phrases that make more sense when paired, like "small town" or "public transportation," it's possible to have the tool treat phrasees as single word by checking the "one word per line" box, and then putting each term (either word or phrase) on a separate line.  Kind of a hassle, though.

**Comment by jridener** [comment]
Here's a word cloud with positive results through row 155 from this Google Sheet: https://docs.google.com/spreadsheets/d/1XdUI2AyVjmd20q8AZlDdrct2d5TfyS3hYai0a6dHaZU/edit?usp=sharing

I heavily edited the words according to @verythorough 's suggestion from 3/30/16. I limited to one word per line to capture multiword ideas like 'San Mateo'.

[wdylawyl_positive_word_cloud_row_155_20160505.zip](https://github.com/opensmc/opensmc.github.io/files/251200/wdylawyl_positive_word_cloud_row_155_20160505.zip)

**Comment by verythorough** [comment]
Nice!  I wonder if we should make a blog post about this?  Could write about (and link) the survey, describe how it was distributed, make a few comments about patterns found and show the word clouds to illustrate.  Any volunteers?

**Comment by jridener** [comment]
Here's a word cloud with negative results through row 155 from this Google Sheet: https://docs.google.com/spreadsheets/d/1XdUI2AyVjmd20q8AZlDdrct2d5TfyS3hYai0a6dHaZU/edit?usp=sharing

The words are heavily edited to match the positive word cloud. I limited to one word per line to capture multi
[wdylawyl_negative_word_clous_row_155_20160506.zip](https://github.com/opensmc/opensmc.github.io/files/252940/wdylawyl_negative_word_clous_row_155_20160506.zip)
word ideas like 'San Mateo'.

**Comment by jridener** [comment]
I'll volunteer write a blog post about this unless someone else wants to: co-authors?

**Comment by verythorough** [comment]
Funny (and not really surprising) how there's overlap between the positive and negative.  I wonder if we should include city names?  Seems a little unfair (and probably inaccurate) to imply, for example, that lots of people like San Carlos, and lots of people dislike Burlingame.

**Comment by jridener** [comment]
Yeah, people are both the best and worst thing about everything, it seems. I was on the fence about including city names but thought it was worth seeing how it worked out. I can take them out and recreate the word clouds.

**Comment by verythorough** [comment]
It was your survey, so do what you like!  Just an observation on my part.

**Comment by jridener** [comment]
I think it's a good point, though. We're already trying to collect where people live in another question. Including whatever locations people mention is an uneven attribution of those issues (or positive mentions) to a place.

**Comment by jridener** [comment]
Negative comments text file through response 155
[Negative comments from what do you love about where you live.txt](https://github.com/opensmc/opensmc.github.io/files/254272/Negative.comments.from.what.do.you.love.about.where.you.live.txt)

**Comment by jridener** [comment]
Positive comments text file through response 155
[Positive comments from what do you love about where you live.txt](https://github.com/opensmc/opensmc.github.io/files/254274/Positive.comments.from.what.do.you.love.about.where.you.live.txt)
